spec/javascripts/modules/track-links_spec.js

```diff
@@ -39,7 +39,7 @@ describe('Track links', function () {
     })
 
     var tracker = new GOVUK.Modules.TrackLinks()
-    tracker.start([element])
+    tracker.init([element])
```

Comment:
There's something odd here, which isn't part of your change but worth raising - this test passes a single element into the module by first wrapping it in an array. The module seems to expect this, acting on the zeroth element of that array (see lines 9 - 11). This is particularly strange as it doesn't do anything if the passes array contains more than one item.

I'm not sure what to suggest here, but it might be worth adjusting the code to see what impact it has on the tests - I can't see a reason why the module is structured like this.

Reply:
no dice I'm afraid. I'll dig into this a bit more and see what I can do!